README.md

```diff
@@ -441,16 +438,16 @@ Note: in case of docker setup, we have provided [some additional scripts](src/se
 Adjust the [remote configs](configs/devnet/remote-setup-config.yaml) and run
 
 ```bash
-../bin/matic-cli setup devnet -c ../configs/devnet/remote-setup-config.yaml
+matic-cli setup devnet --config ../configs/devnet/remote-setup-config.yaml
```

Comment:
can I ask why `-c` and `-i` did not work for you?

Reply:
I was just wondering the meaning of the flags while using `matic-cli` since I'm not used to it. I can also revert this change.

Reply:
No, good one. Let's keep extended for clarity, thanks